pkg/models/models.go

```diff
@@ -30,3 +30,10 @@ type TestGroup struct {
 	Duration  float64
 	Events    []TestEvent
 }
+
+type DatabaseRow struct {
```

Comment:
This should probably be name of the table since u are gonna have multiple tables